main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
+//  Noun: Tape Machine
 //  action 1:
-
+tapeMachine.record();
 //  action 2:
-
+tapeMachine.play();
 //  action 3:
- 
+tapeMachine.rewind();
 //  2)
-//  Noun:
+//  Noun: Player1
 //  action 1:
-
+player1.run();
 //  action 2:
-
+player1.dodge();
 //  action 3:
- 
+player1.jump();
 //  3)
-//  Noun:
+//  Noun: Dog
 //  action 1:
-
+dog.bark();
 //  action 2:
-
+dog.growl();
 //  action 3:
- 
+dog.lick();
 //  4)
-//  Noun:
+//  Noun: Guitar
 //  action 1:
-
+guitar.strum();
 //  action 2:
-
+guitar.tune();
 //  action 3:
- 
+guitar.shred();
 //  5)
-//  Noun:
+//  Noun: Heart
 //  action 1:
-
+heart.pump();
 //  action 2:
-
+heart.speedUpPumping();
 //  action 3:
- 
+heart.slowDownPumping();
 //  6)
-//  Noun:
+//  Noun: Mom
 //  action 1:
-
+mom.callSonToCheckIn();
 //  action 2:
-
+mom.makeFunOfDad();
 //  action 3:
- 
+mom.cookAwesomeDinner();
 //  7)
-//  Noun:
+//  Noun: Drummer
 //  action 1:
-
+drummer.kickDrum();
 //  action 2:
-
+drummer.snareDrum();
 //  action 3:
- 
+drummer.hiHat();
 //  8)
-//  Noun:
+//  Noun: Auto Mixing Class
 //  action 1:
-
+amc.eq();
 //  action 2:
-
+amc.compress();
 //  action 3:
- 
+amc.limit();
```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.